arraycontainer.go

```diff
@@ -439,6 +439,7 @@ func (ac *arrayContainer) lazyorArray(value2 *arrayContainer) container {
 }
 
 func (ac *arrayContainer) and(a container) container {
+	//p("ac.and() called")
```

Comment:
> At a minimum, I think we need to document these printouts. This code gets modified by lots of different people.

I don't know what this would mean. They seem pretty self-evident. What do you have in mind?